idunn/api/places_list.py

```diff
@@ -125,19 +92,6 @@ def valid_raw_filter(cls, v):
                 raise ValueError(f"raw_filter '{x}' is invalid from '{v}'")
         return v
 
-    @validator("category", pre=True, always=True)
-    def valid_category(cls, v):
-        ret = []
-        if not v:
-            return ret
-        for x in v:
-            if x not in ALL_CATEGORIES:
-                raise ValueError(
-                    f"Category '{x}' is invalid since it does not belong to the set of possible categories: {list(ALL_CATEGORIES.keys())}"
-                )
-            ret.append(ALL_CATEGORIES.get(x))
-        return ret
```

Comment:
`pj_source.get_places_bbox` expects `category` to be a list (possibly empty), and breaks with `categories=None`.

Reply:
A small issue remained in the `/v1/instant_answer` implementation that passed `category=None` in some cases, where tests are lacking, my bad.

I pushed a fix on this branch, with an additional test case.